shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
The logical conclusion of this (switching from noun "identity" to noun "login") is that we wind up with "Linked logins" and "allow this login to log in to your account?" on the /account page, which I find more confusing and likely to be seen by a greater number of users.  I find this a worse overall outcome.

Another reason I shy away from "login" as a noun is that we already use it as a verb.

Is there some other word we can use that will work well in all the contexts?  FWIW I don't think we've received criticism of the term "identity" being problematic; only that inconsistency in copy is confusing.